types/google-map-react/index.d.ts

```diff
@@ -66,30 +66,191 @@ declare namespace googleMapReact {
     }
 
     interface Maps {
-        Animation: any;
-        ControlPosition: any;
-        MapTypeControlStyle: any;
-        MapTypeId: any;
-        NavigationControlStyle: any;
-        ScaleControlStyle: any;
-        StrokePosition: any;
-        SymbolPath: any;
-        ZoomControlStyle: any;
-        DirectionsStatus: any;
-        DirectionsTravelMode: any;
-        DirectionsUnitSystem: any;
-        DistanceMatrixStatus: any;
-        DistanceMatrixElementStatus: any;
-        ElevationStatus: any;
-        GeocoderLocationType: any;
-        GeocoderStatus: any;
-        KmlLayerStats: any;
-        MaxZoomStatus: any;
-        StreetViewStatus: any;
-        TransitMode: any;
-        TransitRoutePreference: any;
-        TravelMode: any;
-        UnitSystem: any;
+        Animation: {
+            BOUNCE: number;
+            DROP: number;
+            Uz: number;
+            Qz: number;
+        };
+        ControlPosition: {
+            TOP_LEFT: number;
+            TOP_CENTER: number;
+            TOP: number;
+            TOP_RIGHT: number;
+            LEFT_CENTER: number;
+            LEFT_TOP: number;
+            LEFT: number;
+            LEFT_BOTTOM: number;
+            RIGHT_TOP: number;
+            RIGHT: number;
+            RIGHT_CENTER: number;
+            RIGHT_BOTTOM: number;
+            BOTTOM_LEFT: number;
+            BOTTOM_CENTER: number;
+            BOTTOM: number;
+            BOTTOM_RIGHT: number;
+            CENTER: number;
+        };
+        DirectionsStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            ZERO_RESULTS: string;
+            MAX_WAYPOINTS_EXCEEDED: string;
+            NOT_FOUND: string;
+        };
+        DirectionsTravelMode: {
+            DRIVING: string;
+            WALKING: string;
+            BICYCLING: string;
+            TRANSIT: string;
+            TWO_WHEELER: string;
+        };
+        DirectionsUnitSystem: {
+            METRIC: number;
+            IMPERIAL: number;
+        };
+        DistanceMatrixStatus: {
+            OK: string;
+            INVALID_REQUEST: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            UNKNOWN_ERROR: string;
+            MAX_ELEMENTS_EXCEEDED: string;
+            MAX_DIMENSIONS_EXCEEDED: string;
+        };
+        DistanceMatrixElementStatus: {
+            OK: string;
+            NOT_FOUND: string;
+            ZERO_RESULTS: string;
+        };
+        ElevationStatus: {
+            OK: string;
+            UNKNOWN_ERROR: string;
+            OVER_QUERY_LIMIT: string;
+            REQUEST_DENIED: string;
+            INVALID_REQUEST: string;
+            yz: string;
+        };
```

Comment:
Could not find `yz` in [the documentation](https://developers.google.com/maps/documentation/javascript/reference/elevation#ElevationStatus). Where does it come from ?

Reply:
Please see [main comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60329#issuecomment-1126798899) for response.